src/mono/mono/mini/mini.h

```diff
@@ -2886,6 +2886,18 @@ typedef enum {
 	SIMD_OP_LLVM_I16ABS,
 	SIMD_OP_LLVM_I32ABS,
 	SIMD_OP_LLVM_I64ABS,
+	SIMD_OP_LLVM_I8ABS_SATURATE,
+	SIMD_OP_LLVM_I16ABS_SATURATE,
+	SIMD_OP_LLVM_I32ABS_SATURATE,
+	SIMD_OP_LLVM_I64ABS_SATURATE,
+	SIMD_OP_LLVM_I4FABSOLUTE_COMPARE_GREATER_THAN,
+	SIMD_OP_LLVM_DABSOLUTE_COMPARE_GREATER_THAN,
+	SIMD_OP_LLVM_FABSOLUTE_COMPARE_GREATER_THAN_OR_EQUAL,
+	SIMD_OP_LLVM_DABSOLUTE_COMPARE_GREATER_THAN_OR_EQUAL,
+	SIMD_OP_LLVM_FABSOLUTE_COMPARE_LESS_THAN,
+	SIMD_OP_LLVM_DABSOLUTE_COMPARE_LESS_THAN,
+	SIMD_OP_LLVM_FABSOLUTE_COMPARE_LESS_THAN_OR_EQUAL,
+	SIMD_OP_LLVM_DABSOLUTE_COMPARE_LESS_THAN_OR_EQUAL,
```

Comment:
Hmm, I think calling them SIMD_OP_ARM64 makes more sense since this particular file is not llvm only (although they don't actually get intensified). I think I will fix that in a follow up PR since other ones need to be renamed also.

Reply:
Sure. No problem.